tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1869,6 +1869,23 @@ public void TestColumns()
 
         }
 
+        [Fact]
+        public void TestRows()
```

Comment:
Are there more tests that need to be added here? Ex. setting a value through a DataFrameRow, or enumerating a DataFrameRow.

Reply:
There is now, after we moved to a view. Added them.